packages/donate-button/src/components/EveryMonth/every-month.tsx

```diff
@@ -9,17 +9,20 @@ import Company from 'src/components/Images/Company';
 import {Styled} from 'src/components/Styled';
 import DonationsContext, {AnimationValue} from 'src/contexts/donations-context';
 import OptionsContext from 'src/contexts/options-context';
+import {getFinalOptions} from 'src/helpers/final-options';
 import {
 	DefaultFrequency,
 	DonateButtonOptions,
 	LayoutMode
 } from 'src/helpers/options-types';
 
 interface EveryMonthProps {
-	options: DonateButtonOptions;
+	options: Partial<DonateButtonOptions>;
 	hide: () => void;
 }
 const EveryMonth = ({options, hide}: EveryMonthProps) => {
+	const finalOptions: DonateButtonOptions = getFinalOptions(options);
```

Comment:
ideally we can just rename this in the function to be `options` and the original one to be `inputtedOptions` to make it clear that final is the one to use. seems error prone.